app/components/products/productsService.js

```diff
@@ -0,0 +1,7 @@
+eShopApp.factory('products', function ($http) {
+    return {
+        getProducts: function () {
```

Comment:
the "https://localhost:44302/api" part need to be defined as configuration .. we can add app.config.js file to define all configurations on the application as constants and then inject it here to use the defined configuration values from it